enum/Dialogflow.ts

```diff
@@ -148,3 +148,16 @@ export enum Message {
 	CLOSED_BY_VISITOR = 'Closed by visitor',
 	CUSTOMER_IDLE_TIMEOUT = 'customer_idle_timeout',
 }
+
+export enum CloseChatDescription {
+	CLOSED_BY_VISITOR = 'Closed by visitor',
+	CLOSED_BY_AGENT = 'Closed by agent',
+	CUSTOMER_IDLE_TIMEOUT = 'Customer Timeout',
+	INVALID_TARGET_DEPARTMENT = 'Escalation failed due to invalid target department',
+	SAME_TARGET_DEPARTMENT = 'Escalation failed due to same target department',
+	LIVEAGENT_BOT_OFFLINE_OR_DISABLED = 'Escalation failed due to liveagent bot offline or disabled',
+	NETWORK_OR_APP_ERROR = 'Escalation failed due to network or app error',
```

Comment:
Can we add the App name as well to differentiate between SF or SF app. Because we have a similar event in the SF app.

`Escalation failed due to network or app error in DF app`